torch/distributed/elastic/multiprocessing/errors/error_handler.py

```diff
@@ -83,7 +81,20 @@ def record_exception(self, e: BaseException) -> None:
         JSON format. If the error file cannot be determined, then logs the content
         that would have been written to the error file.
         """
-        _write_error(e, self._get_error_file_path())
+
+        file = self._get_error_file_path()
```

Comment:
why inline this? isn't this the exact same logic as in `_write_error`?

Reply:
to reduce complexity. `_write_error()` was a single line function and it wasn't being used anywhere else.

Reply:
> why inline this? isn't this the exact same logic as in `_write_error`?

oh looks like I published the wrong version, i did actually remove `def _write_error()`